### ISSUE TITLE
Projected CS does not supersede the Geographic Type Geo Key 

### ISSUE THREAD

**Issue by ssfabiopicchi** [laspy/laspy#250]
If we have both the Projected CS and the Geographic Type Geo Key set (redundantly) in a GeoTIFF VLR, the current parse_crs implementation might, for example, output epsg:4326 instead of epsg:32612 if we have las data in UTM 12 North.

I recently stumbled this problem when dealing with a random file we got as input. Here is the line that is causing the bug:
https://github.com/laspy/laspy/blob/dd8a3dada9da228d47f85861ed9fa29199422b1e/laspy/vlrs/known.py#L516

Tomorrow I'll create a PR to handle this situation. Here the geo_keys that triggered the issue:
```
for key in pc2.header.vlrs[0].geo_keys:
...     print(key.id, key.value_offset)
...
1024 1
1025 1
1026 0
2048 4326
2049 64
2050 6326
2052 9001
2054 9102
2056 7030
2057 1
2058 2
2059 0
3072 32612
3073 71
3076 9001
4099 9001
```

**Comment by fabiopicchi** [comment]
Opened the PR with my personal github account:
https://github.com/laspy/laspy/pull/251

**Comment by tmontaigu** [comment]
Closed by #251
